### ISSUE TITLE
Some files are missing

### ISSUE THREAD

**Issue by nareshWcities** [Mohitnt0629/AstronomyPOD#1]
AppDelegate.swift and SceneDelegate.swift missing from project directory
<img width="1394" alt="Screenshot 2023-05-22 at 10 26 06 AM" src="https://github.com/Mohitnt0629/AstronomyPOD/assets/114410036/8e807c1e-d536-4a5e-b9f3-d891888b2608">

**Comment by Mohitnt0629** [comment]
Hello sir

 I have tried to recommit but getting same error let me try again to
resolve this issue mean time I will send you zip file can you please check

**Comment by Mohitnt0629** [comment]
Hello sir,

I have created new repository and commit code working fine now at my end
can you please check


https://github.com/Mohitnt0629/AstronomyPictureOfTheDay
